package.json

```diff
@@ -69,5 +69,8 @@
 			"@typescript-eslint/no-var-requires": "off",
 			"@typescript-eslint/no-empty-function": "off"
 		}
+	},
+	"tsd": {
+		"directory": "type_tests"
```

Comment:
The convention is `test-d`. Then we don't have to specify it here. https://github.com/SamVerschueren/tsd/#test-directory

Reply:
Done.